compose-reference.md

```diff
@@ -124,6 +124,29 @@ shipments:
     enableMonitoring: true
 ```
 
+### healthcheckIntervalSeconds
+
+This value determines how often (in seconds) the platform health check (or liveness probe) is performed. Default is 10 seconds.  *Use caution when increasing this value as it could cause increased application downtime in the case of container crashes.
+
+```yaml
+shipments:
+  my-shipment:
+    env: prod
+    healthcheckIntervalSeconds: 10
+```
+
+### healthcheckTimeoutSeconds
+
+This value determines the number of seconds after which the platform health check (or liveness probe) times out. Default is 1 second.  *Use caution when increasing this value as it could cause increased application downtime in the case of container crashes.
```

Comment:
We can add validation to shipit, but is it worth warning here that interval has to be <= to timeout?  Just in case it saves a little heartburn?  Love the warning though.

Reply:
Yep, I'll add it.